### ISSUE TITLE
Stop demanding location

### ISSUE THREAD

**Issue by thany** [mapsme/omim#12581]
TL;DR - there's no way to have the app not follow the current location.

Here we go again. First reported in [2016](https://github.com/mapsme/omim/issues/4334) and brushed off. Then reported again in [2017](https://github.com/mapsme/omim/issues/6205) and completely ignored. Then again reported in [2017](https://github.com/mapsme/omim/issues/7183) and you basically left it to rot.

Are you guys not able to feel some shame? This is proof that this issue has existed for well over 3 years, and you seem to actively *want* this bug in your app. What's up with that?

How hard is it to fix this bug? Do you need help? Do you need me to tell you what to do? What do you need to get to a fix? Please do tell me, because clearly you're havin problems. This is a 10-minute fix at most, and you're already taking over 3 years.

What needs to happen in order for this bug to get fixed? Please respond.

**Comment by thany** [comment]
Maps.me authors: do not ignore me again, please.

**Comment by thany** [comment]
BLOODY FIX THIS ALREADY WILL YA?
